src/components/AssetSelection/components/AssetMenuButton.tsx

```diff
@@ -55,7 +50,14 @@ export const AssetMenuButton = ({
       isLoading={isLoading}
       {...buttonProps}
     >
-      <Flex alignItems='center' gap={2} {...flexProps}>
+      <Flex
+        alignItems='center'
+        gap={2}
+        width='100%'
+        overflow='hidden'
+        textOverflow='ellisis'
+        whiteSpace='nowrap'
```

Comment:
Did you find any places where it wraps or did you add it as a security? Because it doesn't seem to be even possible, I couldn't find a way to makes words wrapping

Reply:
It was only for additional safely, but you're right it probably isn't necessary - removed in https://github.com/shapeshift/web/pull/7501/commits/91a5ba96663539984423fdd8341d71f9453ba262